### PR TITLE
Implementing Club Registration

### PR DESCRIPTION
**Pull request by danielwei15**
### Summary 

This pull request is the first step towards implementing features for Club Registration. 

- [x] implemented front-end components
- [x] implemented react router and routed registration to url/register
- [ ]  import categories to render dropdown component

### Notes 

Having issues with importing categories in a manner that's consistent and in good practice. Most everything else should be implemented.

### Breaking Changes  

<!-- - Database schema change (anything that changes DynamoDB document structure)
<!-- - Other change that could cause problems (Detailed in notes)

### CODE REVIEW

File: frontend/src/components/ClubRegistration/Dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import './Dropdown.css';
+import { CategoryType } from './categories';
+
+const Dropdown = ({ callback }: { callback: any }) => {
+  // Helper function to convert the strings in the imported enum to human-readable Title Case so no hard-coding is necessary
+  function toTitleCase(str: string) {
```
Comment (20austinw) [inline frontend/src/components/ClubRegistration/Dropdown/Dropdown.tsx:7]: I like your use of helper function and regex here.

File: frontend/src/components/ClubRegistration/Dropdown/categories.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+export enum CategoryType {
```
Comment (20austinw) [inline frontend/src/components/ClubRegistration/Dropdown/categories.tsx:1]: Good naming scheme.

**Review by 20austinw — APPROVED** [review]
Clean code, great stuff! Lgtm.

**Comment by danielwei15** [comment]
Should be merged into main in another PR already.

**Pull request closed without merging**